client/my-sites/plans/jetpack-plans/product-lightbox/quantity-dropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import { isJetpackAISlug } from '@automattic/calypso-products';
+import { SelectDropdown } from '@automattic/components';
+import { useMemo, useCallback, type FC } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLegend from 'calypso/components/forms/form-legend';
+import { getProductPartsFromAlias } from 'calypso/my-sites/checkout/src/hooks/use-prepare-products-for-cart';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import slugToSelectorProduct from '../slug-to-selector-product';
+import useItemPrice from '../use-item-price';
+import { PRODUCT_OPTIONS_HEADER, PRODUCT_TIER_OPTIONS } from './constants';
+import type { SelectorProduct } from '../types';
+
+interface QuantityDropdownProps {
+	product: SelectorProduct;
+	siteId: number | null;
+	onChangeProduct: ( product: SelectorProduct | null ) => void;
+}
+
+const QuantityDropdown: FC< QuantityDropdownProps > = ( { product, siteId, onChangeProduct } ) => {
+	const dispatch = useDispatch();
+	const listPrices = useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
+	const isProductWithTierList = isJetpackAISlug( product.productSlug );
+
+	const tierOptions = useMemo( () => {
+		if ( ! isJetpackAISlug( product.productSlug ) ) {
```

Comment:
Nitpick: could use `isProductWithTierList` here

Reply:
Will get this added and ship 😄

Reply:
Nice catch